app-editors/mg/mg-7.0.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Portable Public Domain Micro Emacs"
+DESCRIPTION="Mg is intended to be a small, fast, and portable editor for \
+people who can't (or don't want to) run emacs for one reason or another, \
+or are not familiar with the vi editor."
+HOMEPAGE="https://github.com/ibara/mg"
+COPYRIGHT="2000"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/ibara/mg/releases/download/mg-$portVersion/mg-$portVersion.tar.gz"
+CHECKSUM_SHA256="650dbdf9c9a72ec1922486ce07112d6181fc88a30770913d71d5c99c57fb2ac5"
+PATCHES="mg-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= 1"
+
+PROVIDES="
+	mg$secondaryArchSuffix = $portVersion
+	cmd:mg$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+        devel:libncursesw$secondaryArchSuffix
+	cmd:make
```

Comment:
Don't think make is required for the devel package (already in BUILD_PREREQUIRES)
```diff
- 	cmd:make
```